packages/components/src/components/as-histogram-widget/utils/data.service.ts

```diff
@@ -0,0 +1,26 @@
+import { max } from 'd3-array';
```

Comment:
why data.service.ts and not data-service.ts?

Reply:
Same with the fragment thingie

Reply:
- Names are separated with hyphens -> semantic meaning
- Separate names with periods -> indicate the type of file (test, service, fragment ...)

Reply:
:+1: 

Now that I think about it, I did the same with fixtures on builder :eyes: